runners/command-runner/src/main/scala/stryker4s/run/Stryker4sArgumentHandler.scala

```diff
@@ -0,0 +1,23 @@
+package stryker4s.run
+import grizzled.slf4j.Logging
+import org.apache.logging.log4j.Level
+import org.apache.logging.log4j.core.config.Configurator
+
+object Stryker4sArgumentHandler extends Logging {
+  private lazy val logLevels: Map[String, Level] =
+    Level.values().map(level => (level.toString.toLowerCase, level)).toMap
+
+  def parseArgs(args: Array[String]): Unit = {
+    // Collect and handle log level argument
+    args
+      .filter(_.head == '-')
+      .map(_.drop(1))
+      .collectFirst { case arg if logLevels.contains(arg) => setLogLevel(logLevels(arg)) }
+      .getOrElse(setLogLevel(Level.INFO))
```

Comment:
To me this whole function is a bit difficult to understand what's going on. The `collectFirst` would make more sense as a `filter` and a `getOrElse` or something. Also, it's more common to start arguments with two dashes (so, --debug). I think it'd also be nice to have the comparison case-insensitive. I'm also missing tests.

Reply:
I refactored it to use `find` instead of `collectFirst`. And also the two dashes and the tests. :)